dist/debian/debian/compat

```diff
@@ -0,0 +1 @@
+9
```

Comment:
What is this?

Reply:
https://www.debian.org/doc/manuals/maint-guide/dother.en.html#compat